kube-controllers/cmd/kube-controllers/main.go

```diff
@@ -121,14 +121,38 @@ func main() {
 	// Create the context.
 	ctx, cancel := context.WithCancel(context.Background())
 
+	// Create the status file. We will only update it if we have healthchecks enabled.
+	s := status.New(statusFile)
+
 	log.Info("Ensuring Calico datastore is initialized")
-	initCtx, cancelInit := context.WithTimeout(ctx, 10*time.Second)
-	defer cancelInit()
-	err = calicoClient.EnsureInitialized(initCtx, "", "k8s")
 	if err != nil {
 		log.WithError(err).Fatal("Failed to initialize Calico datastore")
 	}
 
+	s.SetReady("StartupCalicoDatastoreInitialization", false, "initialized to false")
```

Comment:
can probably just call this "Startup" and then if there are future startup tasks we can clump them together.